src/components/Editor/Breakpoint.js

```diff
@@ -52,7 +53,7 @@ class Breakpoint extends Component<Props> {
     const sourceId = selectedSource.get("id");
     const line = toEditorLine(sourceId, breakpoint.location.line);
 
-    showSourceText(editor, selectedSource.toJS());
+    showSourceText(editor, selectedSource.toJS(), metaData);
```

Comment:
@codehag do we need `showSourceText` after the lifecycle work?

Reply:
I am not sure why this is in addBreakpoint to be honest. this shouldnt be taken care of here -- it should be done with the lifecycle update... in fact the same can be said about everything that is happening with the `editor.codemirror` here. this is outside of the scope of this pr... good catch though

Reply:
ahh ok i understand better. the naming is a bit off -- this should probably be renderBreakpint

Reply:
Filed #4579 and we can come back